outline.config.js

```diff
@@ -29,4 +29,7 @@ module.exports = {
     // default video in any storybook component.
     defaultVideo: 'xiqgG8HUZXE',
   },
+  // Exclude stories by adding the directory they can be found in
+  // ex ['outline-button'] this excludes any story in this directory
```

Comment:
Is this excluding the directories or the files that match? My globbing may be getting confused. :) It looks like it looks for any story files that match the string (outline-button)*.stories.@(js|ts|mdx) in any folder rather than any story files in those folders.

Would that be `../components/**/!(${getExcluded()})/*.stories.@(js|ts|mdx)` with the slash before `*.stories`?

![Glob_Tool___DigitalOcean](https://user-images.githubusercontent.com/397902/152817496-040ab939-679a-49d2-8bca-2ac33cb7cf41.jpg)

![Glob_Tool___DigitalOcean](https://user-images.githubusercontent.com/397902/152817616-19108825-1c8c-44c8-9b72-4c703529ff60.jpg)


https://www.digitalocean.com/community/tools/glob

Reply:
Storybook uses micromatch not glob for their globbing structure, so that's why this syntax looks that way.